modules/virt-booting-efi-mode.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * virt-efi-mode-for-vms.adoc
+
+[id="virt-booting-vms-efi-mode_{context}"]
+= Booting virtual machines in EFI mode
+
+Booting a virtual machine in EFI mode allows for a quicker boot process.
+
+.Procedure
+To boot a VM in EFI Mode:
+
+* Create a .yaml file that defines a VM object or a Virtual Machine Instance (`VMI`) object. Use the firmware section of the example .yaml file:
```

Comment:
Do we refer to these as "yaml files" in general? Not saying this is the wrong term to use. Just wondering if we generally refer directly to the yaml itself.

Also "section" -> "stanza"?

Reply:
Maybe something like:

"Define a VM or Virtual Machine Instance (`VMI`) object with SecureBoot enabled in the firmware stanza"?